build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -131,8 +131,98 @@ steps:
       APP_ROOT=$(agent.builddirectory)/azuredatastudio-linux-x64
       APP_NAME=$(node -p "require(\"$APP_ROOT/resources/app/product.json\").applicationName")
       INTEGRATION_TEST_ELECTRON_PATH="$APP_ROOT/$APP_NAME" \
-      DISPLAY=:10 ./scripts/test-extensions-unit.sh
-    displayName: 'Run Stable Extension Unit Tests'
+      DISPLAY=:10 node ./scripts/test-extensions-unit.js admin-tool-ext-win
```

Comment:
Not a huge fan of the duplication here - it looks like we might be able to loop over the script part and just pass in the test names which seems like a much cleaner solution https://mattvsts.github.io/2019/05/04/solving-the-looping-problem-in-Azure-DevOps-Pipelines/

Reply:
Oh cool! Didn't know they had this!